src/main/java/sirius/web/services/JSONCall.java

```diff
@@ -56,6 +56,22 @@ public static JSONCall to(URL url, String contentType) throws IOException {
         return result;
     }
 
+    /**
+     * Creates a new JSONCall as post request for the given url.
+     *
+     * @param url         the target URL to call
+     * @param contentType the Content-Type to use
+     * @return a new instance to perform the xml call
+     * @throws IOException in case of an IO error
+     */
+    public static JSONCall post(URL url, String contentType) throws IOException {
```

Comment:
why not just markAsPost() to re-use the c'tors above which also set a valid content type